test/blackbox/api/dds-pim/PubSubReader.hpp

```diff
@@ -874,6 +874,44 @@ class PubSubReader
         return *this;
     }
 
+    PubSubReader& set_default_unicast_locators(
+            const eprosima::fastrtps::rtps::LocatorList_t& locators)
+    {
+        participant_qos_.wire_protocol().default_unicast_locator_list = locators;
+        return *this;
+    }
+
+    PubSubReader& add_to_default_unicast_locator_list(
+            const std::string& ip,
+            uint32_t port)
+    {
+        eprosima::fastrtps::rtps::Locator_t loc;
+        IPLocator::setIPv4(loc, ip);
```

Comment:
This assumes `ip` is an ipv4 address, yea?

Reply:
This has been changed in 66cbe857b08ae8a4a471022e82cf41c24944396f to allow both v4 and v6 addresses.